### PR TITLE
Feat/responsive layout

### PR DESCRIPTION
**Pull request by kphutiyagae**
* Added react-content-loader and ant design.
* Created ListCard, ListComponent and HomePage components.
* Merge develop into feat/ui-layout.
* Merge feat/ui-layout into feat/responsive-design.

### CODE REVIEW

**Review by KiveshVirasamy — APPROVED** [review]
Looks good

**Review by MihlaliMgwebi — APPROVED** [review]

File: src/components/UI/atoms/SearchListCard.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export const SearchListCard = ({
+    id,
+    image,
+    title,
+    status,
+    contentRating,
+    styles
+}: ISearchListCardProps) => (
+    <div className={`${styles} border-b-2 flex flex-row mb-3 h-28`}>
+    <img
```
Comment (CharelMarais) [inline src/components/UI/atoms/SearchListCard.tsx:14]: Formatting would make this a bit easier on the eyes.

File: src/components/UI/molecules/ListComponent.tsx [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 
 export const ListComponent = ({
   styles,
+  listData,
 }: IListComponent) => (
-        <List className={`${styles} h-full overflow-auto`}>
+        <List dataSource={listData} className={`${styles} h-full overflow-auto`}
+    
+        >
+
```
Comment (CharelMarais) [inline src/components/UI/molecules/ListComponent.tsx]: Emptiness

File: src/components/pages/HomePage.tsx
Comment (CharelMarais) [inline src/components/pages/HomePage.tsx:1]: Formatting is needed on this page

**Review by CharelMarais — APPROVED** [review]
Looks good, only comments were about some formatting that would help the eyes read the code.

**Pull request merged**